easybuild/easyconfigs/h/h5py/h5py-2.5.0-intel-2016a-Python-2.7.11-HDF5-1.8.16-serial.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = "PythonPackage"
+
+name = 'h5py'
+version = '2.5.0'
+
+homepage = 'http://www.h5py.org/'
+description = """HDF5 for Python (h5py) is a general-purpose Python interface to the Hierarchical Data Format library,
+ version 5. HDF5 is a versatile, mature scientific software library designed for the fast, flexible storage of enormous
+ amounts of data."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+toolchainopts = {'usempi': False}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+hdf5ver = '1.8.16'
+hdf5versuffix = '-serial'
+versionsuffix = '-Python-%%(pyver)s-HDF5-%s%s' % (hdf5ver, hdf5versuffix)
+
+prebuildopts = ' python setup.py configure --hdf5=$EBROOTHDF5 && '
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('HDF5', hdf5ver, hdf5versuffix),
```

Comment:
it's a build dep though, see https://github.com/h5py/h5py/blob/master/setup.py#L35